data/competencies.yml

```
@@ -326,8 +326,11 @@
   domain: null
 
 - id: 2c1289fe-3bf9-476b-981b-8af2d345d76a
-  summary: Implements observability and monitoring when building a solution
-  examples: []
+  summary: Implements appropriate observability and monitoring when building a solution
+  examples: 
+    - When adding a new dependency to a system, adds a healthcheck to monitor the dependency's state
+    - Adds logging that is well-structured and captures useful information about th state of a system
+    - Builds a grafana dashboard that visualises normal and abnormal operation of a system
```

Slightly pedantic change, but 🤷‍♂
```suggestion
    - Builds a Grafana dashboard that visualises normal and abnormal operation of a system
```